examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>May be tf.keras fit method instead of Keras?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>